### PR TITLE
SPU Debugger: SHUFB insertion patterns

### PR DESCRIPTION
**Pull request by elad335**
e.g. from insertion mask creation instructions.

### CODE REVIEW

File: rpcs3/Emu/Cell/SPUDisAsm.cpp [hunk 1]
```diff
@@ -106,6 +147,70 @@ std::pair<bool, v128> SPUDisAsm::try_get_const_value(u32 reg, u32 pc) const
 	return {};
 }
 
+std::tuple<u32, u32, u32> SPUDisAsm::test_insertation_mask(v128 mask)
```
Comment (Nekotekina) [inline rpcs3/Emu/Cell/SPUDisAsm.cpp:150]: Just a nitpick. The problem with tuple - 3 unnamed u32 values. How am supposed to know which one which what?

Comment (VelocityRa) [inline rpcs3/Emu/Cell/SPUDisAsm.cpp:150]: Another nitpick, the word is "insertion".

Comment (elad335) [reply rpcs3/Emu/Cell/SPUDisAsm.cpp]: Fixed.

Comment (elad335) [reply rpcs3/Emu/Cell/SPUDisAsm.cpp]: Fixed.

File: rpcs3/Emu/Cell/SPUDisAsm.h [hunk 2]
```diff
@@ -959,6 +968,20 @@ class SPUDisAsm final : public PPCDisAsm
...
 	{
+		const auto [is_const, value] = try_get_const_value(op.rc);
+
+		if (is_const)
+		{
+			const auto [size, dst, src] = try_get_insert_mask_info(value);
+
+			if (size)
+			{
+				// Comment insertation pattern
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/SPUDisAsm.h:979]:
```suggestion
				// Comment insertion pattern
```

Comment (elad335) [reply rpcs3/Emu/Cell/SPUDisAsm.h]: Fixed.

**Pull request merged**